### PR TITLE
Feature/feature pet attributes refactor

### PR DESCRIPTION
**Pull request by marclll**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WeightData.java  -18
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Weight.java  -15
         

Complexity increasing per file
==============================
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Wash.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Training.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5480909&bid=17961960)

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClient.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import java.util.concurrent.ExecutionException;
 
 /**
- * @author Oriol Catalán
+ * @author Marc Simó
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClient.java:22]: You are not the creator of this class

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/GenderType.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package org.pesmypetcare.usermanagerlib.datacontainers;
 
 /**
- * @author Santiago Del Rey
+ * @author Marc Simó
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/GenderType.java:4]: You are not the creator of this class

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Pet.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import java.util.Map;
 
 /**
- * @author Santiago Del Rey
+ * @author Marc Simó
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Pet.java:12]: You are not the creator of this class

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/PetData.java [hunk 4]
```diff
@@ -6,7 +6,7 @@
 import java.util.Map;
 
 /**
- * @author Santiago Del Rey
+ * @author Marc Simó
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/PetData.java:9]: You are not the creator of this class

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * @author Marc Simó
+ */
+public class TrainingData {
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java:9]: All the methods are missing their javadoc

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * @author Marc Simó
+ */
+public class WashData {
```
Comment (santidrj) [inline usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java:9]: All the methods are missing their javadoc

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java (hunk 5 above)
Comment (marclll) [reply usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java:9]: This class will be changed so I will comment it once it's finished

File: usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java (hunk 6 above)
Comment (marclll) [reply usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java:9]: This class will be changed so I will comment it once it's finished

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/WashData.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/TrainingData.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Training.java  2
- usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Wash.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5480909&bid=17961960)

**Pull request merged**